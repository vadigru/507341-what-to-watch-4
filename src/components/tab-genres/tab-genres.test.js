import React from "react";
import renderer from "react-test-renderer";
import TabCatalog from "./tab-genres.jsx";

const className = ``;

const mockProps = {
  tabName: `tabName`,
  activeTab: `activeTab`,
  onTabClick: () => {}
};

it(`Should render TabCatalog component`, () => {
  const tree = renderer
    .create(
        <TabCatalog
          className={className}
          {...mockProps}
        />)
  .toJSON();

  expect(tree).toMatchSnapshot();
});
