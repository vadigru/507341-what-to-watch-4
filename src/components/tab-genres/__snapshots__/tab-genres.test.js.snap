// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render TabCatalog component 1`] = `
<li
  className="catalog__genres-item false"
>
  <a
    className="catalog__genres-link"
    href="#"
    onClick={[Function]}
  >
    tabName
  </a>
</li>
`;
